servers/rendering/renderer_rd/effects_rd.cpp

```diff
@@ -1889,6 +1926,21 @@ void EffectsRD::sort_buffer(RID p_uniform_set, int p_size) {
 }
 
 EffectsRD::EffectsRD(bool p_prefer_raster_effects) {
+	{
+		Vector<String> FSR_upscale_modes;
+
+		if (RD::get_singleton()->get_device_capabilities()->supports_fsr_upscale_normal) {
```

Comment:
Maybe "default" would be a better word than "normal" in order to not confuse it with normals.

Reply:
Maybe, but _normal_ is used by AMD to describe the half floating point compute shader

Reply:
I changed `supports_fsr_upscale_normal` to `supports_fsr_upscale_half_float`